.github/workflows/static_page_verification.yml

```diff
@@ -0,0 +1,56 @@
+name: Cloud Formation Template Static Page verification
+
+on:
+  push: {branches: dev}
+  pull_request: {branches: dev} 
+  
+jobs:
+  cft-static-page-chk:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - name: Install Dependencies
+        run: |
+          python -m pip install --user boto3
+          python -m pip install --user requests
+          pip install git-remote-codecommit
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{secrets.AWS_ACCESS_KEY_ID}}
+          aws-secret-access-key: ${{secrets.AWS_SECRET_ACCESS_KEY}}
+          aws-session-token: ${{secrets.AWS_SESSION_TOKEN}}
+          aws-region: ${{secrets.AWS_REGION}}
+      - name: Create Stack with CF template
+        uses: aws-actions/aws-cloudformation-github-deploy@v1
+        with:
+          name: nap-cft-stack
+          template: "/home/runner/work/aws-waf-solution-template/aws-waf-solution-template/modules/quickstart/nap-autoscale-ubuntu-dev.yaml"
+          parameter-overrides: "sshKey=NAP_AWS,uniqueString=CftUnq"
+          capabilities: CAPABILITY_NAMED_IAM
+      - name: assuming that the configured files are under repo scripts folder  
```

Comment:
Comment doesn't seem to match content. Are you waiting till the stack gets ready?

Reply:
No , after the stack is created... Here I'm manually creating the files into aws codecommit repo with static configuration for that I have given this sleep, once the files are copied under code base we can remove these lines of code.

Reply:
Why you don't just copy all necessary files along with appspec.yaml (line 39)?